### ISSUE TITLE
global post unable to starred/watch/favorite

### ISSUE THREAD

**Issue by jasoncheng** [jasoncheng/mawa-android#115]
networkpublic_timeline post cannot add to favorite
always return 400 error

**Comment by jasoncheng** [comment]
https://github.com/friendica/friendica/issues/6305

**Comment by annando** [comment]
Yeah, I guess this is currently not possible, due to the database structure.

**Comment by jasoncheng** [comment]
@annando good morning, and thanks for reply; 
if you're available, please teach me on friendica database schema; currently i only know, 

each public network post will have user=0 AND global=1, 
and each user will have his own item too.

anyway, thanks for you hard work on Friendica
